changelog-template.hbs

```diff
@@ -0,0 +1,20 @@
+{{#each releases}}
+
+  ## [{{title}}] {{niceDate}}
+  {{#if summary}}
+    {{summary}}
+  {{/if}}
+
+  {{#each merges}}
+    - {{#if commit.breaking}}**Breaking change:** {{/if}}{{message}}
+  {{/each}}
+
+  {{#each fixes}}
+    - {{#if commit.breaking}}**Breaking change:** {{/if}}{{commit.subject}}{{#each fixes}}{{/each}}
```

Comment:
Is the nested `{{#each fixes}}{{/each}}` here intended?

Reply:
negative.